front-end/src/components/main-page/TranscriptDescription.jsx

```diff
@@ -0,0 +1,26 @@
+/* istanbul ignore file */
+import React, { Component } from 'react';
+import {exampleTranscript} from "../helpers/ExampleTranscript.js"; 
+
+class TranscriptDescription extends Component {
+	render() {
+		return (
+            <div className='whitespace-nowrap'>
+                <p className='text-center font-bold'> 
+                    Required Data Format:
+                </p>
+                <pre className='overflow-auto leading-8 text-sm'>
+                    {JSON.stringify(exampleTranscript, null, 2)}
+                </pre>
+                <br/> 
+                <p> 
+                    * Your file can contain up to 180 questions.
+                    <br/> 
+                    * Questions must be chonologically ordered.
+                </p>
```

Comment:
I like the styling on everything, but just in this part with the paragraphs, I think that more padding could be added to the questions since they're closer to the right edge of the rectangle, maybe consider playing around with the tailwind here (either in a new div or straight-up inside the paragraph classname) to do maybe a justify-center or a pr-[number] (p standing for padding, r standing for right). Tailwind docs and messing around is your best friend!